virtool/startup.py

```diff
@@ -193,6 +194,10 @@ async def init_fake(app: aiohttp.web_app.Application):
         await virtool.dev.fake.populate(app)
 
 
+async def init_faker(app: aiohttp.web_app.Application):
+    app["faker"] = virtool.faker.VirtoolFaker()
```

Comment:
I have been using `from` imports lately because they work better with Pycharm.

```suggestion
    app["faker"] = VirtoolFaker()
```